webpack/components/AnsibleRolesSwitcher/AnsibleRolesSwitcherActions.js

```diff
@@ -45,6 +52,30 @@ export const getAnsibleRoles = (
     .catch(error => dispatch(errorHandler(ANSIBLE_ROLES_FAILURE, error)));
 };
 
+export const getAnsibleVariables = (url, search, resourceName, resourceId, parentId) => dispatch => {
+  dispatch({ type: ANSIBLE_VARIABLES_REQUEST });
+
+  if (!search.search) {
```

Comment:
We want only variables for assigned roles. If there is no search, there are no roles assigned.